static/js/services/enketo.js

```diff
@@ -24,47 +23,30 @@ angular.module('inboxServices').service('Enketo',
     var FORM_ATTACHMENT_NAME = 'xml';
     var REPORT_ATTACHMENT_NAME = this.REPORT_ATTACHMENT_NAME = 'content';
 
-    var replaceJavarosaMediaWithLoaders = function(formInternalId, form) {
```

Comment:
The main change is by passing the docId instead of the internalId we can look up the attachment without first querying for the docId. That removed a bunch of code and error handling. This is outside of the caching code so will be executed after the HTML is retrieved from the cache and cloned.